utils/apxdx.F90

```diff
@@ -21,6 +21,10 @@
 !> @author J. Ator @date 2009-07-01
 program apxdx
 
+#ifdef NAGFOR
```

Comment:
We really don't want compiler specific ifdefs in the code. Is there another way we can resolve this?

What is the problem with exit?

Reply:
The problem is that exit is not a standard Fortran intrinsic, so compilers don't have to provide it.

Reply:
Maybe STOP and/or ERROR STOP would be a standardly correct alternative?  They can return error codes to the environment.